portal/config/eproms/Questionnaire.json

```diff
@@ -7843,6 +7843,17 @@
           "valueBoolean": true
         }
       ]
+    },
+    {
+      "identifier": [
+        {
+          "system": "http://us.truenth.org/questionnaire",
+          "value": "ironman_ss_post_tx"
+        }
+      ],
+      "status": "draft",
+      "resourceType": "Questionnaire",
+      "item": []
```

Comment:
@pbugni  am I correct in recalling that I should populate this?

Reply:
Done @pbugni

Reply:
looks great, thank you @mcjustin